srv-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+
+services:
+  db:
+    image: ${POSTGRES_IMAGE}:${POSTGRES_VERSION}
+    environment:
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+  omero:
+    build: ..
+    environment:
+      - ROOTPASS=${ROOTPASS}
+      - CONFIG_omero_db_user=postgres
+      - CONFIG_omero_db_name=postgres
+      - CONFIG_omero_db_pass=${POSTGRES_PASSWORD}
+      - CONFIG_omero_upgrades_url=
+      - CONFIG_Ice_IPv6=0
+      - PYTHONPATH=/opt/omero/server/OMERO.server/lib/python
+    ports:
+      - "${OMERO_SERVER_TCP}:4063"
```

Comment:
``:`` has been dropped from docker-compose.
Shouldn't it be the case here too?

Reply:
Ah, good catch. Yes, thanks.